### PR TITLE
Stuck web entities

### PR DESCRIPTION
**Pull request by sethalves**
- fix a bug that was causing non-physical held things to get stuck to avatar's hand

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: c4706490766f8fb5bf0531536db4f12c4e2328e1 available at: https://gist.github.com/f575521d3eae72c6fd9e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7158.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7158.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: f113a83c5b2844905fd099d1c1deacce22ba0358 available at: https://gist.github.com/f575521d3eae72c6fd9e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7158.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7158.dmg

File: examples/controllers/handControllerGrab.js [hunk 1]
```diff
@@ -1589,7 +1589,10 @@ function MyController(hand) {
         ids.forEach(function(id) {
             var props = Entities.getEntityProperties(id, ["boundingBox", "name"]);
             if (props.name === 'pointer') {
-                return;
+                continue;
+            }
+            if (!props || !props.boundingBox) {
+                continue;
```
Comment (Atlante45) [inline examples/controllers/handControllerGrab.js]: Can you use `continue` here? Since it's a function, feels to me like you would have to return, no?

**Comment by Atlante45** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: f113a83c5b2844905fd099d1c1deacce22ba0358 available at: https://gist.github.com/f575521d3eae72c6fd9e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7158.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7158.dmg

File: examples/controllers/handControllerGrab.js [hunk 2]
```diff
@@ -1589,7 +1589,10 @@ function MyController(hand) {
         ids.forEach(function(id) {
             var props = Entities.getEntityProperties(id, ["boundingBox", "name"]);
             if (props.name === 'pointer') {
-                return;
+                continue;
+            }
+            if (!props || !props.boundingBox) {
```
Comment (jherico) [inline examples/controllers/handControllerGrab.js]: you check `!props` after you use `props.name`.  The `!props` check should be moved above the `props.name` access.

Comment (Atlante45) [inline examples/controllers/handControllerGrab.js]: everything could be in a single clause even.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 23b204c0c6390fd2e5a9d95a20649da7c0e6f6d5 available at: https://gist.github.com/f575521d3eae72c6fd9e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7158.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7158.dmg

**Pull request merged**